sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -314,12 +315,17 @@ public abstract partial class Request : System.IDisposable
     public abstract partial class RequestContent : System.IDisposable
     {
         protected RequestContent() { }
+        public static Azure.Core.RequestContent Create(System.BinaryData content) { throw null; }
```

Comment:
`+` Tests for new method here.

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/tests/HttpPipelineRequestContentTests.cs